fetch_api/src/fetch_api/arm.py

```diff
@@ -76,6 +76,19 @@ def __init__(self):
         self._move_group_client.wait_for_server(rospy.Duration(10))
         self._compute_ik = rospy.ServiceProxy('compute_ik', GetPositionIK)
         self._tf_listener = TransformListener()
+        self.move_group = MoveGroupInterface(ARM_GROUP_NAME, "base_link")
+
+    """
+    Tucks arm
+    """
+    def tuck(self):
+        joints = ["shoulder_pan_joint", "shoulder_lift_joint", "upperarm_roll_joint",
+                  "elbow_flex_joint", "forearm_roll_joint", "wrist_flex_joint", "wrist_roll_joint"]
+        pose = [1.32, 1.40, -0.2, 1.72, 0.0, 1.66, 0.0]
+        while not rospy.is_shutdown():
+            result = self.move_group.moveToJointPosition(joints, pose, 0.02)
```

Comment:
Unless I'm missing something that the Interface class gives, this should just be passed to `move_to_joint_goal`